packages/web-components/src/components/va-link-action/va-link-action.tsx

```diff
@@ -74,7 +74,7 @@ export class VaLinkAction {
 
   private linkRef?: HTMLElement;
 
-  componentDidLoad() {
+  componentWillLoad() {
```

Comment:
Updating the lifecycle here will resolve this warning:

![Screenshot 2024-11-18 at 2 51 57 PM](https://github.com/user-attachments/assets/124b5de7-75d5-414a-86d7-f8c9d5877cbb)